cmd/cluster-agent/app/app.go

```diff
@@ -189,6 +193,9 @@ func start(cmd *cobra.Command, args []string) error {
 		custommetrics.StopServer()
 	}
 	clusterAgent.Stop()
+	if stopCh != nil {
+		close(stopCh)
```

Comment:
I think we will never reach this because `<-signalCh` is blocking. Let's investigate the stopping signals for the DCA

Reply:
The idea is for the `stopCh` to be closed after receiving `SIGTERM` from the OS to do a graceful shutdown.